### ISSUE TITLE
Stellaris 3.7.2 Canis Mainor tech tree

### ISSUE THREAD

**Issue by SamueleDecarli** [turanar/stellaris-tech-tree#31]
We got new techs and primitive insights with Stellaris 3.7.2. It would be nice to get an updated tech tree for them.

**Comment by HellFAF** [comment]
Indeed, now with 3.8 out and huge changes to leaders it may be worth revisiting the files to see if things changed.
Do you guys need any help with this? I am also digging into files to update Tech Tree Exposed with the new techs added since 3.5 so let me know if you need help!
